### ISSUE TITLE
Change shakelib.rupture to impactutils.rupture

### ISSUE THREAD

**Issue by cbworden** [shakemap-ci/shakemap-aqms#4]
This is an issue in the shakemap repo that should have been over here. I'm going to try to link them.

https://github.com/usgs/shakemap/issues/1229

**Comment by dutchTender** [comment]
use the following solution to correct python module path issue.
@cbworden  just to confirm. we are to make the following 2 changes.


modify the shakemap-aqms .py files that call for import Origin,
aqms_db2xml.py orig:
from shakelib.rupture.origin import Origin
new:
from impactutils.rupture.origin import Origin

aqms_eq2xml.py orig:
from shakelib.rupture.origin import write_event_file
new:
from impactutils.rupture.origin import write_event_file

**Comment by cbworden** [comment]
Yes, Paul said this worked for him. All of you should fork, clone, and start following this repo so that these issues are known to the group. Let's see if we can get everyone subscribed.

**Comment by dutchTender** [comment]
sounds good.

**Comment by dutchTender** [comment]
to whom it might interest, use above step as a workaround until the fix for this issue is merged into the main master repo.
